DEVELOPMENT.md

```diff
@@ -251,38 +245,53 @@ The folder structure of the frontend looks as follows:
 ├── template.go       go template module used to render the frontend HTML
 ```
 
+#### Interactive development stack launcher tool
+
+In order to easily launch development environments in different deployment contexts, this mage target configures and starts a development environment for The Things Stack, allowing users to choose between local and staging environments, enable branding, and configure cloud-hosted mock setups. You can launch it via:
+
+```bash
+$ tools/bin/mage dev:serveDevWebui
+```
+
+It will interactively guide you through the desired setup and launches the The Things Stack Enterprise as well as a frontend development server wia webpack.
+
 #### Development Configuration
 
-> To use a convenient interactive launcher for the development environments without any further setup required, please see the (interactive development stack launcher tool)[#interactive-development-stack-launcher-tool]
+> The preffered way to set up the development environment is to use the [interactive development stack launcher tool](#interactive-development-stack-launcher-tool).
 
 In order to set up The Things Stack to support running the frontend via `webpack-dev-server`, the following environment setup is needed:
```

Comment:
So all these env blocks are only needed when _not_ using `dev:serveDevWebui`, right?

If that is the case, can this be moved lower in the page for advanced use?